tests/integration/cmd_dev_test.go

```diff
@@ -1488,7 +1555,7 @@ var _ = Describe("odo dev command tests", func() {
 			checkFunc         func(stdout, stderr string)
 		}
 		testForCmd := func(tt testCase) {
-			err := helper.RunDevMode(tt.devAdditionalOpts, func(session *gexec.Session, outContents []byte, errContents []byte, ports map[string]string) {
+			err := helper.RunDevMode(nil, tt.devAdditionalOpts, func(session *gexec.Session, outContents []byte, errContents []byte, ports map[string]string) {
```

Comment:
I think the two arguments are inverted. That might explain why the corresponding tests are no longer passing.

```suggestion
			err := helper.RunDevMode(tt.devAdditionalOpts, nil, func(session *gexec.Session, outContents []byte, errContents []byte, ports map[string]string) {
```